src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_subdirectory(client_backend)
 set(
   PERF_ANALYZER_SRCS
   main.cc
+  perf_analyzer_runner.cc
```

Comment:
had to move contents of old main.cc to new file so that those functions/variables could be used, but also allow two main()s to exist--one for Perf Analyzer binary, one for unit test runner.

If the old main.cc was left as is, then the unit test runner, having its own main, wouldn't be able to compile because the other source files require some functions/variables in the original main.cc.

The new main.cc is just a shell to call some function that does everything the old main.cc does. This allows both the unit test runner and the perf analyzer binary to compile with those definitions but not have multiple main functions

Reply:
perf_analyzer_runner.c -> perf_analyzer.cc

Reply:
Renamed 👍